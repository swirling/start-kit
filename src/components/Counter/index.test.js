import React from 'react';
import renderer from 'react-test-renderer';
import { Counter } from './index';

it('renders counter', () => {
  const tree = renderer.create(<Counter a="1" count={3} />).toJSON();
  expect(tree).toMatchSnapshot();
});
