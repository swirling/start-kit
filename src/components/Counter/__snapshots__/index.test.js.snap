// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders counter 1`] = `
<div
  className="a"
>
  <div
    className="b"
  >
    3
    ----
    1
  </div>
  <button
    onClick={[Function]}
  >
    addone
  </button>
</div>
`;
